x-pack/test/api_integration/apis/beats/enroll_beat.js

```diff
@@ -22,18 +20,22 @@ export default function ({ getService }) {
     let beat;
 
     beforeEach(async () => {
-      validEnrollmentToken = chance.word();
+      validEnrollmentToken =
+        'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.' +
+        'eyJjcmVhdGVkIjoiMjAxOC0wNi0zMFQwMzo0MjoxNS4yMzBaIiwiaWF0IjoxNTMwMzMwMTM1fQ.' +
+        'SSsX2Byyo1B1bGxV8C3G4QldhE5iH87EY_1r21-bwbI';
```

Comment:
does this mean the enrollment token (the one the user has to copy&paste) will look like this?